src/lib/account_update.ts

```diff
@@ -1288,6 +1289,12 @@ class AccountUpdate implements Types.AccountUpdate {
       other
     );
   }
+  static emptyValue() {
+    let empty = Types.AccountUpdate.emptyValue();
+    empty.body.authorizationKind.verificationKeyHash =
+      dummyVerificationKeyHash();
+    return empty;
+  }
```

Comment:
oof, this makes `AccountUpdate` differ from the auto-generated one. It seems fine as a quick fix, but medium term I'd want to add an annotation in OCaml which allows us to specify a custom `emptyValue()` at the lower level where it also ends up in the autogenerated type. We already do the same for `ActionState`

Reply:
Gotcha! I was trying to figure out how to override `emptyValue` but couldn't see how it was done. I'll take a look for `ActionState` :+1: